### DISCUSSION TITLE
Error accessing Maven central from ubuntu-20.04 runner

### DISCUSSION THREAD

**Discussion by SanjayVas** [community/community#53258] [Actions]
### Select Topic Area

Bug

### Body

My workflows using the GitHub-hosted ubuntu-20.04 runner suddenly cannot access Maven central.

```
Caught java.net.UnknownHostException (repo.maven.apache.org) while downloading https://repo.maven.apache.org/maven2/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.6.21/kotlin-stdlib-jdk8-1.6.21.pom
```

This started happening ~30 minutes ago.

**Comment by tarun-neem** [comment]
I'm having the same issue as well.

**Reply by justinwb** [reply]
Can confirm the same problem

**Reply by kxuenvoy** [reply]
similiar issue here.

**Reply by hgschmie** [reply]
same here.

**Reply by tuteng** [reply]
Same here, the mvn repository seems to be pointing to a non-existent host https://repo.maven.apache.org/maven2

**Reply by justinwb** [reply]
I submitted a bug report about 20 minutes ago. Might be worth others doing the same to bump the priority in addition to the posts here.

**Reply by tarun-neem** [reply]
@justinwb, URL please

**Reply by yesjm** [reply]
me too

**Reply by aaronvolkmann** [reply]
same

**Reply by DenisBurmistrov** [reply]
same, so sad

**Reply by sjchoRsquare** [reply]
got the same error

  > Could not resolve org.springframework.boot:spring-boot-dependencies:3.0.2.
             > Could not get resource 'https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/3.0.2/spring-boot-dependencies-3.0.2.pom'.
                > Could not GET 'https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/3.0.2/spring-boot-dependencies-3.0.2.pom'.
                   > repo.maven.apache.org: Name or service not known

**Reply by jhkang1517** [reply]
same error ...

> Could not GET 'https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.2/spring-boot-dependencies-2.6.2.pom'.

**Reply by cnx-keganonomichi** [reply]
I'm having the same problem

**Reply by tuteng** [reply]
Add host record temporary fix this issue -> https://github.com/orgs/community/discussions/53262#discussioncomment-5668167

```
  - name: Add hosts to /etc/hosts
    run: |
      sudo echo "146.75.124.215 repo.maven.apache.org" | sudo tee -a /etc/hosts
```

**Reply by cnx-brunomoretto** [reply]
if you are trying to build a docker image add the flag to docker build --add-host=repo.maven.apache.org:146.75.124.215

**Reply by kwonjael** [reply]
Just Gradle Build ( Don't Use Docker Image )
Same Error~~!! OMG!!!!

>          > Could not resolve org.springframework.boot:spring-boot-dependencies:2.5.7.
>             > Could not get resource 'https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.5.7/spring-boot-dependencies-2.5.7.pom'.
>                > Could not GET 'https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.5.7/spring-boot-dependencies-2.5.7.pom'.
>                   > repo.maven.apache.org: Name or service not known

**Reply by imed-pedrosantos** [reply]
> if you are trying to build a docker image add the flag to docker build --add-host=repo.maven.apache.org:146.75.124.215

it worked for me

**Reply by nguyendanghoangdao** [reply]
> Add host record temporary fix this issue -> https://github.com/orgs/community/discussions/53262#discussioncomment-5668167
> 
> ```
>   - name: Add hosts to /etc/hosts
>     run: |
>       sudo echo "146.75.124.215 repo.maven.apache.org" | sudo tee -a /etc/hosts
> ```

Thanks, it worked for me

**Reply by kwonjael** [reply]
KST 2023-04-20 13:40 
This issue has been resolved

**Comment by imed-pedrosantos** [comment]
same error.

Could not transfer artifact org.springframework.boot:spring-boot-starter-parent:pom:3.0.3 from/to central (https://repo.maven.apache.org/maven2):

**Comment by cnx-joaosilva** [comment]
same error here

**Comment by raphael5178** [comment]
same error here Unknown host repo.maven.apache.org: Name does not resolve

**Comment by cnx-alexduarte** [comment]
Oh no!

Same Error Here!!! 

Failed to read artifact descriptor for org.jacoco:jacoco-maven-plugin:jar:0.8.4: Could not transfer artifact org.jacoco:jacoco-maven-plugin:pom:0.8.4 from/to central (https://repo.maven.apache.org/maven2): transfer failed for https://repo.maven.apache.org/maven2/org/jacoco/jacoco-maven-plugin/0.8.4/jacoco-maven-plugin-0.8.4.pom: Unknown host repo.maven.apache.org: Name or service not known -> [Help 1]

**Comment by DenisBurmistrov** [comment]
Same

Could not transfer artifact org.springframework.boot:spring-boot-dependencies:pom:2.5.6 from/to central (https://repo.maven.apache.org/maven2): transfer failed for https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.5.6/spring-boot-dependencies-2.5.6.pom @ line 30, column 16: Unknown host repo.maven.apache.org: Name or service not known

**Comment by dmshin1305** [comment]
OMG. SAME ERROR HERE
Could not GET 'https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.7.0/spring-boot-dependencies-2.7.0.pom'.

**Comment by samar19947** [comment]
fakyu adhhab limudajaeat amik

احصل على Outlook لـ iOS<https://aka.ms/o0ukef>
________________________________
من: 신동민 ***@***.***>
‏‏تم الإرسال: Thursday, April 20, 2023 4:40:32 AM
إلى: community/community ***@***.***>
نسخة: Subscribed ***@***.***>
‏‏الموضوع: Re: [community/community] Error accessing Maven central from ubuntu-20.04 runner (Discussion #53258)


OMG. SAME ERROR HERE
Could not GET 'https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.7.0/spring-boot-dependencies-2.7.0.pom'.

**Comment by samus-aran** [comment]
Hi y'all 👋 

Thank you for supporting each other! @tuteng & @cnx-brunomoretto kudos to you two for supplying workarounds 👏

We believe the issue was with the central Maven repository, in which their DNS has not fully completed propagated. Unfortunately this issue was outside of GitHub's control. You'll need to wait for the DNS propagation to be completed, the runners should be able to reach the Maven repository then. I believe we have reports that confirms this has started to be resolved 👍
